### DISCUSSION TITLE
Perfomance improvement cheatsheet for AIO instance.

### DISCUSSION THREAD

**Discussion by brunoalfred** [nextcloud/all-in-one#1334] [Questions] [answered]
It's very important to understand tips on how to monitor performance of a running AIO instance. Please highlight necessary things to consider when one feels like the nextcloud instance doesn't perform as expected regardless of infrastructure resource  being enough.

**Comment by szaimen** [comment]
I guess I could write dome some general suggestions but it is probably more useful to get into the details:

What exactly is slow on your instance, what is the available hardware, how many users do you have, which options are activated?

**Reply by brunoalfred** [reply]
Currently we have 5-10 users, with not more that 5 concurrent sessions. 
The hardware resources are: OS unix -- RAM 16 GB CPU cores -- 16.

The challenges is the application takes much time to load and process requests.

**Reply by brunoalfred** [reply]
![Screenshot 2022-10-28 at 14 52 16](https://user-images.githubusercontent.com/55545250/198580973-0c9d05a5-a102-4685-b86c-bbea75342847.png)

**Comment by szaimen** [comment]
See https://github.com/nextcloud/all-in-one/discussions/1335 for general recommendations

**Comment by szaimen** [comment, marked as answer]
@brunoalfred please follow the general recommendations: https://github.com/nextcloud/all-in-one/discussions/1335. 

if all is fullfilled and it is still slow, you might run into the same problem like described in https://github.com/nextcloud/all-in-one/discussions/1300 which triggers the brute-force protection and throttles the requests.
